AutoRaise.mm

```diff
@@ -377,27 +376,27 @@ - (void) readConfig: (int) argc {
 
 - (void) readOriginalConfig {
     // original config files:
-    NSString *delayFile = @"AutoRaise.delay";
-    NSString *warpFile  = @"AutoRaise.warp";
-    
-    if ([self fileExist: delayFile] or [self fileExist: warpFile]) {
-        NSFileHandle *hDelayFile = [NSFileHandle fileHandleForReadingAtPath: [self getFilePath: delayFile]];
+    NSString *delayFilePath = [self getFilePath: @"AutoRaise.delay"];
+    NSString *warpFilePath = [self getFilePath: @"AutoRaise.warp"];
+
+    if (delayFilePath || warpFilePath) {
+        NSFileHandle *hDelayFile = [NSFileHandle fileHandleForReadingAtPath: delayFilePath];
```

Comment:
I'm afraid it might not work in the right way.
The 'NSString to BOOL' in objective-c works in a slightly unexpected way for me.
But this has to be tested in practice. I must have been doing something wrong. :/

Reply:
See below:

    int * integerPointer = NULL; // NULL is a pointer that points nowhere
    int anInteger;
    integerPointer = &anInteger; // now integerPointer points to anInteger

So integerPointer can point to anInteger object somewhere in memory or it can point nowhere.
That fact that a pointer can point nowhere, I am using in the code above (NULL evaluates to false)

Reply:
@Dimentium 
> I must have been doing something wrong.

I think you were using the wrong operator, bitwise or, instead of logical ||
https://www.techotopia.com/index.php/Objective-C_Operators_and_Expressions#Bitwise_OR

Reply:
I think I understand how it works with Integers, but with NSString it is slightly different - not null string == false like NULL.

Reply:
> not null string == false like NULL

Not sure if I understand this. The behaviour of a pointer is always the same, regardless of where it points to (NSString, int, float, whatever). The code below will output both log lines:

    if ("not null") { NSLog(@"I was here"); }
    NSString * str = @"not null";
    if (str) { NSLog(@"I was here too"); }

A bit off topic, but while I was testing, the above I did notice this strange behaviour:
    
    NSString * p1 = @"str";
    NSString * p2 = @"str";
    if (p1 == p2) { NSLog(@"I am not showing, which is expected"); }
    if (@"str" == @"str") { NSLog(@"I am showing, which is unexpected"); }

For this last statement, the compiler issues a warning however.

Reply:
@Dimentium 
Ah, and testing even more and combining this with the information in your PR #38, I think I understand:

    NSString * pStr = NULL;
    if ([pStr isEqual: NULL]) {
        NSLog(@"I am not showing!");
    }

The reason for this is that isEqual looks at the contents of the string, but since pStr points nowhere, there is no contents to look at and it will return false. On the other hand the following will show the log line as expected:

    NSString * pStr = NULL;
    if (pStr == NULL) {
        NSLog(@"I am showing, like I should");
    }

Reply:
@sbmpost 
Oh! Now it is more clear for me.
Thank you again! )